com.google.EarthPro.yaml

```diff
@@ -0,0 +1,66 @@
+app-id: com.google.EarthPro
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+sdk: org.freedesktop.Sdk
+command: earth-runner
+separate-locales: false
+finish-args:
+  - --share=network
+  - --share=ipc
+  - --socket=x11
+  - --socket=pulseaudio
+  - --device=dri
+  - --persist=.googleearth
+  - --filesystem=xdg-download
+  - --talk-name=org.freedesktop.Notifications
+cleanup:
+  - /include
+  - /lib/pkgconfig
+  - /lib/*.a
+modules:
+  - name: libglu
+    buildsystem: meson
+    sources:
+      - type: archive
+        url: https://mesa.freedesktop.org/archive/glu/glu-9.0.2.tar.xz
+        sha256: 6e7280ff585c6a1d9dfcdf2fca489251634b3377bfc33c29e4002466a38d02d4
+        x-checker-data:
+          type: anitya
+          project-id: 13518
+          stable-only: true
+          url-template: https://mesa.freedesktop.org/archive/glu/glu-$version.tar.xz
+
+  - name: earth
+    buildsystem: simple
+    build-commands:
+      - mkdir -p deb-package
+      - ar p earth.deb data.tar.xz | tar -xJf - -C deb-package
+      - mv deb-package/opt/google /app/google
+      - install -Dm755 earth-runner /app/bin/earth-runner
+      - install -Dm644 /app/google/earth/pro/google-earth-pro.desktop /app/share/applications/com.google.EarthPro.desktop
+      - install -Dm644 /app/google/earth/pro/product_logo_256.png /app/share/icons/hicolor/256x256/apps/com.google.EarthPro.png
+      - install -Dm644 /app/google/earth/pro/product_logo_128.png /app/share/icons/hicolor/128x128/apps/com.google.EarthPro.png
+      - install -Dm644 /app/google/earth/pro/product_logo_64.png /app/share/icons/hicolor/64x64/apps/com.google.EarthPro.png
+      - install -Dm644 /app/google/earth/pro/product_logo_48.png /app/share/icons/hicolor/48x48/apps/com.google.EarthPro.png
+      - install -Dm644 /app/google/earth/pro/product_logo_32.png /app/share/icons/hicolor/32x32/apps/com.google.EarthPro.png
+      - install -Dm644 /app/google/earth/pro/product_logo_16.png /app/share/icons/hicolor/16x16/apps/com.google.EarthPro.png
+      - install -Dm644 com.google.EarthPro.xml /app/share/metainfo/com.google.EarthPro.metainfo.xml
+      - desktop-file-edit --remove-key=MultipleArgs /app/share/applications/com.google.EarthPro.desktop
+      - desktop-file-edit --set-icon=com.google.EarthPro /app/share/applications/com.google.EarthPro.desktop
+      - desktop-file-edit --set-key=Exec --set-value='earth-runner' /app/share/applications/com.google.EarthPro.desktop
+    sources:
+      - type: file
```

Comment:
Unless the licence allows redistribution, this has to be reworked to use `extra-data`.

Reply:
Implemented